lib/stripped-build-plugins.js

```diff
@@ -16,24 +16,29 @@ module.exports = function() {
   //     features[feature] = false;
   //   }
   // }
```

Comment:
I guess we can finally remove this comment too, as it is not relevant anymore...